### ISSUE TITLE
Mobile data check: Windows

### ISSUE THREAD

**Issue by programmin1** [programmin1/Repeater-START#15]
An equivalent to Linux NetworkManager should be available to check if on wifi/ethernet, or, some connected hotspot/satellite/4g that likely is metered and not something to download the latest repeaters off of. As of now this option isn't in settings for Windows branch.

https://github.com/programmin1/Repeater-START/pull/13

**Comment by RamNisanth** [comment]
is this issue still open? can I take up this issue ?

**Comment by programmin1** [comment]
Yes, I have not started on this. For starters you can check out _windows_ branch and branch off of that.

Will you join us for #Hacktoberfest next month?

**Comment by RamNisanth** [comment]
sure!

**Comment by RamNisanth** [comment]
I am interested

**Comment by karshav16** [comment]
Would love to work on this issue. Kindly Assign!

**Comment by programmin1** [comment]
Go ahead and work on a start on the issue you know most about if you can - thank you.

**Comment by karshav16** [comment]
This issue is part of hacktoberfest right ?

**Comment by karshav16** [comment]
Isn't this part of hacktober fest? if so, would request you to assign this
issue to me!

**Comment by programmin1** [comment]
Sure, unless @RamNisanth has progress on this issue to share so far?
Why don't you both pick one of the issues, see how far you can go on finding a solution, we can review on Wednesday zoom meeting.
I am not familiar with Windows network/api.
